### PR TITLE
Meta: Move hotfixes to new repo + allow string hotfixes

### PR DESCRIPTION
**Pull request by fregante**
- As mentioned in https://github.com/refined-github/refined-github/issues/5222#issuecomment-1167741255
- See the content in https://github.com/refined-github/yolo
- Note that when creating commits on the new repo, issue references **in commit titles** need to be `refined-github/refined-github#1234` instead of just `#1234` (or just the whole issue URL)

### CODE REVIEW

File: source/features/index.tsx [hunk 1]
```diff
@@ -323,8 +333,8 @@ void add('rgh-deduplicator' as FeatureID, {
 	async init() {
 		// `await` kicks it to the next tick, after the other features have checked for 'has-rgh', so they can run once.
 		await Promise.resolve();
-		select('#js-repo-pjax-container, #js-pjax-container')?.append(<has-rgh/>);
-		select('#repo-content-pjax-container, turbo-frame')?.append(<has-rgh-inner/>); // #4567
+		select(_`#js-repo-pjax-container, #js-pjax-container`)?.append(<has-rgh/>);
+		select(_`#repo-content-pjax-container, turbo-frame`)?.append(<has-rgh-inner/>); // #4567
```
Comment (fregante) [inline source/features/index.tsx:337]: This is likely to change soon, so it's the first target for dynamic replacement

File: source/features/index.tsx [hunk 2]
```diff
@@ -103,6 +105,8 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		getLocalStrings(),
 	]);
 
+	console.log({value: _`Original`});
```
Comment (fregante) [inline source/features/index.tsx]: Line 98 and line 108 show the regular string and then the string after the replacement

Screen Shot 1

File: source/helpers/hotfix.ts [hunk 3]
```diff
@@ -16,21 +18,30 @@ function parseCsv(content: string): string[][] {
...
 
 	// Rate-limit check
+	if (content) {
+		return atob(content).trim();
+	}
+
+	return '';
+}
+
+export type HotfixStorage = Array<[FeatureID, string]>;
```
Comment (fregante) [inline source/helpers/hotfix.ts:35]: I think eventually we should have a single `updater` function and a single `hotfixes` object in the options, like:

```js
{
	style: 'body {color: red}',
  broken: ['forked-to'],
  strings: {from: 'to'},
}
```

File: source/helpers/hotfix.ts [hunk 4]
```diff
@@ -16,21 +18,30 @@ function parseCsv(content: string): string[][] {
...
+
+export const updateHotfixes = cache.function(async (version: string): Promise<HotfixStorage> => {
+	const content = await fetchHotfix('broken-features.csv');
 	if (!content) {
 		return [];
 	}
 
 	const storage: HotfixStorage = [];
-	for (const [featureID, unaffectedVersion, relatedIssue] of parseCsv(atob(content))) {
+	for (const [featureID, relatedIssue, unaffectedVersion] of parseCsv(content)) {
```
Comment (fregante) [inline source/helpers/hotfix.ts:44]: The order has been updated:

https://github.com/refined-github/yolo/blob/main/broken-features.csv


Screen Shot 4

Comment (jpe-io) [reply source/helpers/hotfix.ts:44]: FS

**Review by jpe-io — COMMENTED** [review]
Hi

**Review by JessiG2319 — APPROVED** [review]

**Pull request merged**